### ISSUE TITLE
Consider not showing option to apply an effect from chat card when that effect has transfer enabled

### ISSUE THREAD

**Issue by schultzcole** [foundryvtt/dnd5e#2984]
With the changes from #2935, the way I see it, the `transfer` property on an active effect ("Apply Effect to Actor" in the active effect UI) indicates that it is an inherent property of the item being _owned by_ the actor, not an effect that can be _granted_ by using the item.

To that end, it seems reasonable to me that there would be essentially no circumstances when you would want to be able to give a `transfer` effect to another actor/player, and the source actor can just toggle it on or off from the effects tab of the sheet since it is transferred, so there is no reason to want to be able to apply effects of that kind when that item is displayed in chat.

This would help to cement the distinction between `transfer` effects and non-`transfer` effects:

- `transfer` effects show on the owning actor's sheet and can be toggled on and off as appropriate, but cannot be granted through usage of the item.
- non-`transfer` effects can only be given to an actor by using the item (or manually dragging the effect from the item to the actor).

**Comment by Fyorl** [comment]
Yes, it will be important to streamline our implementation here and remove any special cases like this when we start adding more complicated logic like duration expiration and concentration. I think we're aligned on applying effects via the effects tray being the mechanism for automation here, so the self-cast single-target exceptions like *Shield* or *Bless* should have `transfer: false` and only be applied via the effects tray. If users prefer the old way of toggling these effects they can still set `transfer: true`.

**Comment by krbz999** [comment]
yoink
